ToDo.DataAccess/ToDo.DataAccess/ToDoRepository.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using ToDo.DataAccess.DataBase;
+using ToDo.DataAccess.Models;
+
+namespace ToDo.DataAccess
+{
+    public class ToDoRepository : IDataRepository
+    {
+        private readonly ToDoDbContext _context;
+
+        ToDoRepository(ToDoDbContext newContext)
```

Comment:
Rename parameter `newContext` to `context`

Reply:
renamed.